### PR TITLE
Remove spinner on Give gold page

### PR DESCRIPTION
**Pull request by jo-asakura**
:eyeglasses: @ajacksified or @danehansen

### CODE REVIEW

File: src/views/pages/userGild.jsx [hunk 1]
```diff
@@ -48,8 +48,8 @@ class UserGildPage extends React.Component {
     var token = this.props.token;
     var user = this.props.user || {};
 
-    var userProfile = this.state.data.data || {};
-    var name = this.props.userName;
+    //var userProfile = this.state.data.data || {};
+    //var name = this.props.userName;
```
Comment (ajacksified) [inline src/views/pages/userGild.jsx]: delete these lines?

**Comment by ajacksified** [comment]
:nail_care:

File: src/views/pages/userGild.jsx (hunk 1 above)
Comment (jo-asakura) [inline src/views/pages/userGild.jsx]: They're not mine.. I commented them out since functionality is not completed. If you don't need them I'll remove it.

**Comment by ajacksified** [comment]
:fish:

**Pull request closed without merging**